integrations/gcp/.port/spec.yaml

```diff
@@ -5,8 +5,19 @@ features:
     section: Cloud Providers
 saas:
   enabled: false
-configurations: []
+configurations:
+  - name: encodedADCConfiguration
+    type: string
+    require: false
+    description: Base64 encoding of your Google Cloud's ADC Credentials configuration file. Follow https://cloud.google.com/docs/authentication/provide-credentials-adc to learn more.
```

Comment:
can we try be more proactive and actually provide a step by step guide or at least an example. 

e.g. 
`echo "~/adc.json" | base64 | pbcopy`